examples/php-features-test/test-all.sh

```
@@ -0,0 +1,262 @@
+                if find "$output_dir" -name "*ServiceClient.php" | grep -q .; then
+                    print_success "Found client files"
+                fi
+                if ! find "$output_dir" -name "*ServiceInterface.php" | grep -q .; then
+                    print_success "No server interfaces (client-only mode)"
+                fi
+                ;;
+        esac
+        
+        # Check namespace in generated files
+        if [ -f "$output_dir"/*.php ] || find "$output_dir" -name "*.php" | head -1 > /dev/null; then
+            local sample_file=$(find "$output_dir" -name "*.php" | head -1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)